controllers/dependencies.go

```diff
@@ -83,11 +89,11 @@ func (r *OperatorPipelineReconciler) removePipelineDependencyFiles(filePath stri
 	return nil
 }
 
-func (r *OperatorPipelineReconciler) applyManifests(fileName string, Namespace string, obj client.Object) error {
+func (r *OperatorPipelineReconciler) applyPipelineManifests(fileName string, meta metav1.ObjectMeta, obj *tekton.Pipeline) error {
 
 	b, err := os.ReadFile(fileName)
 	if err != nil {
-		log.Error(err, "Couldn't read manifest file")
+		log.Error(err, "Couldn't read manifest file for ")
```

Comment:
It would be great to print the `fileName`

Reply:
done in both methods.